OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalDbContract.java

```diff
@@ -66,7 +66,7 @@
       public static final String COLUMN_NAME_TIMESTAMP = "timestamp";
       public static final String COLUMN_NAME_NAME = "name";
       public static final String COLUMN_NAME_SESSION = "session";
-      public static final String COLUMN_NAME_PARAMS = "params";
+      public static final String COLUMN_NAME_WEIGHT = "weight";
```

Comment:
params was made, because on the future it will be able to save strings, bundles, and other types, so it was a generic param for extension purposes

Reply:
We decided it over complicates the functionality for now since it only has 1 attribute. We can always re-add it in the future, but for now we should keep it as simple as possible for easier usage and understanding.

The main reason is we now return the `OutcomeEvent` in the `onSuccess` callback and a `getParams` would be very confusing for the end user and they would need to know the keys inside the `JSON` to get the weight of the `OutcomeEvent`